frontend/src/components/main/user/pending_invites.jsx

```diff
@@ -1,74 +1,34 @@
-import React from 'react'
-import InviteItem from './invite_item';
-
-
-
-class PendingInvites extends React.Component {
-  constructor(props){
-    super(props)
-
-    this.state={
-      pendingInvites : "",
-    }
-
-    this.checkNotEmpty = this.checkNotEmpty.bind(this);
-
-  }
-
+import React from 'react';
+import { useDispatch } from 'react-redux';
 
-  checkNotEmpty(object) {
-    for (const key in object) {
-      if (object.hasOwnProperty(key))
-        return true;
-    }
-    return false;
-  }
-
-
-  componentDidMount(){
-    this.props.requestUser(this.props.currentUser.id);
-  }
-
-
-  componentDidUpdate(){
-    if (this.checkNotEmpty(this.props.users)) {
-
-      let pendingInvites = this.props.users[this.props.currentUser.id] ?
-        this.props.users[this.props.currentUser.id].pendingInvites : null;
+import InviteItem from './invite_item';
 
-      if (pendingInvites) {
-        this.state['pendingInvites'] = pendingInvites;
-      }
-    }
-  }
-  
-  render(){
 
-    let pending;
-    pending =  this.state.pendingInvites ? 
-      <ul>
-      {Object.values(this.state.pendingInvites).map(pending=>{
-        if(pending)
-        {return(
-          <InviteItem
-          key={pending.id}
-          currentUser = {this.props.users[this.props.currentUser.id]}
-          pending={pending}
-          updateUser={this.props.updateUser}
-          addUserToCommunity={this.props.addUserToCommunity}
-          />
-        )}
+export default ({currentUser, requestUser, updateUser, users, addUserToCommunity}) => {
+  const dispatch = useDispatch();
```

Comment:
I don't think you're ever actually using this dispatch. It seems that you should be able to do without it

Reply:
Oh yep, I'll remove that